manager/resource.go

```diff
@@ -15,11 +15,11 @@
 package manager
 
 import (
-v1 "k8s.io/api/core/v1"
-"k8s.io/apimachinery/pkg/labels"
-corev1listers "k8s.io/client-go/listers/core/v1"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	corev1listers "k8s.io/client-go/listers/core/v1"
 
```

Comment:
Why is there a newline here?

Reply:
Probably goimports did something it shouldn't have done. Let me fix that.

Reply:
Fixed